api/types/types.go

```diff
@@ -327,7 +327,7 @@ type ContainerJSONBase struct {
 	Name            string
 	RestartCount    int
 	Driver          string
-	Platform        string
+	OS              string
```

Comment:
Why the rename?

Reply:
@mlaventure It's due to this that I put at the top:

In addition, it renames (almost all) uses of the string variable platform (and associated
methods/functions) to os, or operatingSystem where it clashes with the os import. This makes it much clearer to disambiguate with the swarm "platform" which is really os/arch. This is a stepping stone to getting the daemon towards fully multi-platform/arch-aware, and makes it clear when "operating system" is being referred to rather than "platform" which is misleadingly used - sometimes in the swarm
meaning, but more often as just the operating system.

Reply:
In this case, shouldn't we use an OCI platform spec?

Reply:
Ah, yes, indeed. `Platform` was added much earlier in the LCOW work. This PR simply renamed it to `OS`. But in the light of the discussions about the OCI spec and specifiers, I agree, this should be the OCI image-spec `Platform`. Will update.

Reply:
@stevvooe - That's updated now. For the time being, only the OS field is populated as that the only field in the `Container` object currently. As the daemon is moved to be fully multi-platform/arch (etc) aware, as per the F2F discussions, this field can be extended.